code-reviews.md

```diff
@@ -0,0 +1,141 @@
+# Code Review Guide
+
+> A friendly guide for reviewing code--and not each other--at CFPB.
+
+## Why reviews?
+
+Code reviews are incredibly important step to the development process. Our goals
+when reviewing code are to:
+
+- expose bugs before they make it to production
+- ensure a concistent quality in the codebase
+- create an environment for sharing knowledge and developing skills
+- encourage open communication between the entire team
+
+
+## Tips for a successfull review
+
+For a code review to be successfull, we all need to be on the same page. To help
+accomplish this, everyone should follow these tips.
+
+### For everyone
+
+- Remember that we're reviewing the code, not the author. Nothing should be
+  given or taken taken personally.
+- There is often more than one way to approach a solution. Discuss tradeoffs and
+  reach a resolution quickly.
+- Refer to the style guides whenever there's a discussion that can't be resolved.
+- Ask questions rather than make statements. ("What do you think about...?")
+- Ask for clarification if the code or comments are unclear. ("I didn't understand.
+  Can you clarify?")
+- Avoid selective ownership of code. ("mine", "not mine", "yours")
+- Avoid using terms that could be seen as referring to personal traits.
+- Be explicit, people don't always understand your intentions online.
+- Be humble. ("I'm not sure - let's look it up.")
+- Don't use hyperbole. ("always", "never", "endlessly", "nothing")
+- Don't use sarcasm.
+- Keep it real. If emoji, animated gifs, or humor aren't you, don't force them.
+  If they are, use them with aplomb.
+- Talk offline if there is too much back and forth. Post a follow-up comment
+  summarizing the discussion.
+- Praise team members when they create exemplary work or suggestions.
+
+### For code submitters
+
+- Link to the code review from the originating Jira task/issue, if applicable.
+- Remember that the code isn't you, don't get defensive when a reviewer is critical
+  of the code.
+- Seek to understand the reviewer's perspective.
+- Try to respond to every comment.
+- Be grateful for the reviewer's suggestions. ("Good catch, fixing in a4994ec")
+- Explain why the code exists. ("We need to work around these existing patterns")
+- Extract out-of-scope changes and refactorings into future Jira tasks/issues.
+- Push commits based on earlier rounds of feedback as isolated commits to the
+  branch. Do not squash until the branch is ready to merge. Reviewers should be
+  able to read individual updates based on their earlier feedback.
+- Wait to merge the branch until it has passed Continuous Integration testing.
```

Comment:
This implies that the person who submits the pull requests also merges it. I know Flapjack does it that way, but eRegs doesn't. How do other teams at CFPB handle merging PRs?